marketplacetests/tests/test_marketplace_login_during_purchase.py

```diff
@@ -31,4 +32,4 @@ def test_purchase_app(self):
         # Switch back to the Marketplace frame and wait for the install button to update
```

Comment:
We should complete the create/confirm pin workflow which should return us to the expected screen.